src/vm/moar/runner/main.c

```diff
@@ -123,13 +125,11 @@ int retrieve_home(char *out_home, char *rel_home, char *env_var, char *exec_dir_
 
     check_file_path = (char*)malloc(home_size + 50);
     memcpy(check_file_path, out_home, home_size);
-    strcpy(check_file_path + home_size, check_file);
-    if (!file_exists(check_file_path)) {
-        free(check_file_path);
-        return 0;
-    }
+    strncpy(check_file_path + home_size, check_file, 50);
```

Comment:
srncpy does not add a trailing \0 when it runs into the limit. We should do that manually

Reply:
Oh, right. I was getting it confused with `strlcpy`, which does as long as the destination buffer has room, not the source